Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -203,3 +203,29 @@ This path can be retrieved from the Environment API, see :ref:`Environment-label
     :hidden:
 
     LegacyInstalations
+
+
+.. _directory-vendor:
+
+:file:`vendor/`
+===============
+
+In this directory, which lies outside of the webroot, all extensions (system,
+third party and local) are installed as composer packages.
+
+The directory contains folders for each required vendor and inside each
+vendor directory there is a folder for the composer name.
```

Comment:
```suggestion
vendor directory there is a folder with the different project names.
```
There is no "composer name".

See: https://getcomposer.org/doc/01-basic-usage.md#package-names